### ISSUE TITLE
After command doesn't work on file change

### ISSUE THREAD

**Issue by mikegleasonjr** [tockins/realize#75]
on OSX

```
    - type: after
      command: <whatever>
      changed: true
      startup: true
```

Doesn't work on file change, only when quitting.

If I change "after" for "before", it is working on file change.

**Comment by Asoseil** [comment]
#94  with v1.5.0 there are commands global and commands to be executed on each project refresh

**Comment by amirkheirabadi** [comment]
I have this the problem too.
what time you are ready for merge and fix this?
thankyou

**Comment by Asoseil** [comment]
this will be definitely fixed with v2.0, expected for the next week
